### PR TITLE
Feature/quassel glep 81

### PR DESCRIPTION
**Pull request by johu**

### CODE REVIEW

File: acct-user/quassel/quassel-0.ebuild [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+ACCT_USER_ID=472
+ACCT_USER_HOME=/var/lib/quassel
```
Comment (mgorny) [inline acct-user/quassel/quassel-0.ebuild]: Do you need this, given that init.d hardcodes the path anyway?

Comment (johu) [reply acct-user/quassel/quassel-0.ebuild]: I just migrated the existing function to create the cert of pkg net-irc/quassel

Comment (mgorny) [reply acct-user/quassel/quassel-0.ebuild]: I don't know how this is relevant. FWICS, the function also uses hardcoded path.

Comment (a17r) [reply acct-user/quassel/quassel-0.ebuild]: note that I haven't done a GLEP81 ebuild so far; isn't `/var/lib/<username>` the default anyway, and it is recommendet not to be set in that case?

Comment (mgorny) [reply acct-user/quassel/quassel-0.ebuild]: `/dev/null` is the default, and it's recommended to leave it as that.

Comment (johu) [reply acct-user/quassel/quassel-0.ebuild]: Where is the difference between acct-user/git with  www-apps/gitea for example to quassel? So should I use a designed eclass function to create the dir with the correct user and group or should I do it in ebuild?

**Pull request closed without merging**